js/kucoinfutures.js

```diff
@@ -1095,7 +1095,106 @@ module.exports = class kucoinfutures extends kucoin {
             'margin': amount,
             'bizNo': uuid,
         };
-        return await this.futuresPrivatePostPositionMarginDepositMargin (this.extend (request, params));
+        const response = await this.futuresPrivatePostPositionMarginDepositMargin (this.extend (request, params));
+        //
+        //    {
+        //        code: '200000',
+        //        data: {
+        //            id: '62311d26064e8f00013f2c6d',
+        //            symbol: 'XRPUSDTM',
+        //            autoDeposit: false,
+        //            maintMarginReq: 0.01,
+        //            riskLimit: 200000,
+        //            realLeverage: 0.88,
+        //            crossMode: false,
+        //            delevPercentage: 0.4,
+        //            openingTimestamp: 1647385894798,
+        //            currentTimestamp: 1647414510672,
+        //            currentQty: -1,
+        //            currentCost: -7.658,
+        //            currentComm: 0.0053561,
+        //            unrealisedCost: -7.658,
+        //            realisedGrossCost: 0,
+        //            realisedCost: 0.0053561,
+        //            isOpen: true,
+        //            markPrice: 0.7635,
+        //            markValue: -7.635,
+        //            posCost: -7.658,
+        //            posCross: 1.00016084,
+        //            posInit: 7.658,
+        //            posComm: 0.00979006,
+        //            posLoss: 0,
+        //            posMargin: 8.6679509,
+        //            posMaint: 0.08637006,
+        //            maintMargin: 8.6909509,
+        //            realisedGrossPnl: 0,
+        //            realisedPnl: -0.0038335,
+        //            unrealisedPnl: 0.023,
+        //            unrealisedPnlPcnt: 0.003,
+        //            unrealisedRoePcnt: 0.003,
+        //            avgEntryPrice: 0.7658,
+        //            liquidationPrice: 1.6239,
+        //            bankruptPrice: 1.6317,
+        //            settleCurrency: 'USDT'
+        //        }
+        //    }
+        //
+        const data = this.safeValue (response, 'data');
+        return this.parseMargin (data, market);
```

Comment:
@kroitor @frosty00 

I feel like we should lean away from doing stuff like this. The user already has this information because they passed it to the method, and providing it in the response makes it misleading that the exchange returned this information, and can make bugs more difficult to find (if for example, the exchange is returning no information, but we provide information in the response)

Reply:
imho, responses are not only for submitters, as they might be used for further processing, and as long as we return 'success' response, we might already know that exchange executed what we have requested. so, unified responses are not solely meaning that exchange returned those info (i.e. another subject we are talking today: `marginMode: add/reduce` - exchange doesnt return that responses, but we add them in unified responses because of completeness). 
idk, i think so

Reply:
> imho, responses are not only for submitters, as they might be used for further processing, and as long as we return 'success' response, we might already know that exchange executed what we have requested. so, unified responses are not solely meaning that exchange returned those info (i.e. another subject we are talking today: `marginMode: add/reduce` - exchange doesnt return that responses, but we add them in unified responses because of completeness). idk, i think so

a success response doesn't always mean it worked, sometimes the exchange returns an empty response or something

Reply:
as long as 'status' is not 'complete/ok', i think filling other fields with `this.extend` wouldn't hurt, but idk, that's just my thought.